hedy.py

```diff
@@ -1220,6 +1220,8 @@ def repeat(self, args):
         var_name = self.get_fresh_var('i')
         times = process_variable(args[0], self.lookup)
         command = args[1]
+        if command.startswith("print"):
```

Comment:
That's true. In my mind it was because there was no need to add a sleep for operations that didn't involve output, since the kids won't notice any change, just the next print will execute slower, for example:
 ```
i is 1
repeat 3 times i is i+3
print i
```
So I thougth about just a sleep in the cases where the repeat consists only of `print` statements. Also the issue said: "When using a repeat with only prints the text appears at once" so I thought it was just about prints. It was my mistake, I didn't thought about loops in other levels. Sorry for that!